service/src/main/kotlin/fi/espoo/evaka/pis/controllers/PartnershipsController.kt

```diff
@@ -98,7 +102,12 @@ class PartnershipsController(private val asyncJobRunner: AsyncJobRunner, private
                 val partnership = partnershipService.updatePartnershipDuration(tx, partnershipId, body.startDate, body.endDate)
                 asyncJobRunner.plan(
                     tx,
-                    listOf(NotifyFamilyUpdated(partnership.partners.last().id, partnership.startDate, partnership.endDate))
+                    listOf(
+                        GenerateFinanceDecisions.forAdult(
+                            partnership.partners.last().id,
```

Comment:
Could there be another way to ensure we generate the decision for the correct partner than the partners.last()? I know it is obvious which partner it is when looking at the implementation of partnershipService.createPartnership, but there is no guarantee in the API that the order is maintained when creating the parnership. Anyway, left for author's consideration whether this can be improved or not.

Reply:
Well to be honest it doesn't matter which adult is picked, the end result should be the same. I'm not sure why it was `last()` originally. I'll look into this.